packages/manager/apps/telecom/src/app/telecom/telephony/line/translations/Messages_fr_FR.json

```diff
@@ -16,5 +16,7 @@
   "telephony_line_expire_date": "Le service expire le {{date}}.",
   "telephony_line_expire_link": "Vous pouvez annuler en cliquant ici.",
   "telephony_line_convert_date": "Cette ligne sera convertie en numéro le {{ date }}.",
-  "telephony_line_convert_link": "Vous pourrez annuler dans la section suivante."
+  "telephony_line_convert_link": "Vous pourrez annuler dans la section suivante.",
+  "telephony_line_mgcp_banner_link": "Cliquez ici.",
+  "telephony_line_mgcp_banner_description": "Le terminal MGCP utilisé est déprécié : veuillez effectuer un renouvellement de terminal afin de le remplacer."
```

Comment:
I have the impression that the sentence is redundant.
Can be seen to edit with product owner: "Le terminal MGCP utilisé est déprécié : veuillez effectuer un remplacement de terminal."